Examples/ExpenseManagementUseCaseExample/app/src/main/java/uk/fidel/kotlinstreamexample/MainActivity.kt

```diff
@@ -0,0 +1,76 @@
+package uk.fidel.kotlinstreamexample
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.toBitmap
+import com.fidelapi.Fidel
+import com.fidelapi.entities.CardScheme
+import com.fidelapi.entities.FidelResult
+import com.fidelapi.entities.ProgramType
+import com.fidelapi.entities.abstraction.OnResultObserver
+import org.json.JSONException
+import org.json.JSONObject
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        Fidel.programId = "your program id"
+        /**
+         * For a real project, it is recommended to store the SDK key on your backend and retrieve it
+         * before starting to use the SDK, after the user authenticates.
+         */
+        Fidel.sdkKey = "pk_test_your_sdk_key"
+        Fidel.supportedCardSchemes =
+            setOf(CardScheme.VISA, CardScheme.MASTERCARD, CardScheme.AMERICAN_EXPRESS)
```

Comment:
Since Stream only supports Visa, I think the following suggestion makes sense, just to avoid confusion.

```suggestion
            setOf(CardScheme.VISA)
```